modules/ROOT/pages/exchange-api.adoc

```diff
@@ -259,6 +260,50 @@ https://anypoint.mulesoft.com/exchange/api/v2/assets/:groupId/:assetId/:version/
   }'
 ----
 
+[[update-mutable-data]]
+=== Update Mutable Data
```

Comment:
This is a task topic title; did you mean to include steps?

Reply:
The existing subsections under "Describe Your Assets" all use the imperative form to describe an action ("Add Tags", "Add Categories", "Add Custom Fields") and then show an example that performs the action, with brief explanation about the example. I've reworded the new section "Update Asset Metadata" to make sure it matches exactly.